sashimi/hardware/light_source/interface.py

```diff
@@ -23,25 +24,31 @@ def __init__(self, port):
     def start(self):
         pass
 
+    @abstractmethod
     def set_power(self, current):
         """Sets power of laser based on self.intensity and self.intensity_units"""
         pass
 
+    @abstractmethod
     def close(self):
         pass
 
     @property
+    @abstractmethod
     def intensity(self):
         return None
 
     @intensity.setter
+    @abstractmethod
     def intensity(self, exp_val):
         pass
```

Comment:
hm there is power, intensity and current, I guess we should make the `set_current` private in cobolt and communicate the units with other properties.

Reply:
I endorse this

Reply:
I agree, I'll create a new Issue and PR to address this specifically